### DISCUSSION TITLE
DaisyUI theme has no defaultLight color and seems to always use the first registered theme

### DISCUSSION THREAD

**Discussion by AetherDim** [saadeghi/daisyui#2947] [Q&A] [answered]
For my application I would like to select a light and dark theme which is independend of the theme load order.

If I change the daisyUI config to look like this:
themes: ["dark", "light", "cupcake"],

The theme defaults to dark and does not honor the system default.

Is it possible to add a llightTheme setting in addition to the darkTheme setting?

https://play.tailwindcss.com/6t9GZ65waM

"daisyui": "4.9.0",

**Comment by saadeghi** [comment, marked as answer]
Use `themes: ["light", "dark", "cupcake"],`  
It will respect the OS/browser darkmode by default.

Because regardless of daisyUI, by default any web page is light mode. If OS is set ot dark mode we can set a dark mode preference in CSS to respect the system settings and that's what everyone does (as well as daisyUI)

Let me know if you have any questions.

**Reply by AetherDim** [reply]
Thankes for the quick reply!

For my application, I have no real control over the list order because it is dynamically generated. For now, I search for the light theme and append it to the front of the list.

For the dark theme I just use the daisyUI setting.

Is my assumention correct that daisyUI will use the first theme as light theme?

**Reply by saadeghi** [reply]
There is a default theme (first one) it can be light or anything else. It's up to you.

And if the browser or OS is set to dark mode, we apply the dark theme using `prefers-color-scheme: dark`
